test/WebJobs.Script.Tests/TestScripts/FSharp/DocumentDBIn/run.fsx

```diff
@@ -16,9 +16,9 @@ open System
 open Microsoft.Azure.WebJobs.Host
 open Newtonsoft.Json.Linq
 
-[<CLIMutable>]
-type QueueInput =
-    { DocumentId : string }
+//[<CLIMutable>]
+type QueueInput() =
+    member val DocumentId : string = "" with get,set
```

Comment:
Note to @tomaspetricek and myself: One or both of the changes to DocumentDbIn and DocumentDbOut caused the DocumentDb tests to start passing. So something is sensitive to records v. ``member val``.